gatsby-config.js

```diff
@@ -1,10 +1,14 @@
 require(`dotenv`).config({
   path: `.env`,
-})
+});
```

Comment:
I think the initial standard was to not have semicolons. Let's keep that (or if you feel like updating all js files to have semicolons, feel free to 🤷‍♂)